_posts/0000-01-02-Oamusan0913.md

```diff
@@ -0,0 +1,6 @@
+_ _ _
+layout: slide
+title: "welcome to our second slide!"
+_ _ _ 
+Your text
```

Comment:
"God is good all the time"
[My_CV.txt](https://github.com/Oamusan0913/github-slideshow/files/3720985/My_CV.txt)